### DISCUSSION TITLE
Extra status line font is too big

### DISCUSSION THREAD

**Discussion by cvancar** [NightscoutFoundation/xDrip#2020] [Q&A]
I just installed the a7f75ab build and realized that the font of the Extra Status Line is too big. Is it expected behavior? Google Pixel 4a 5G, Android 11

<img width="350" src="https://user-images.githubusercontent.com/6818543/155936968-7c1cbdb7-7d59-4fb1-8a61-3c16a4420c87.png" />

**Comment by Navid200** [comment]
Does this happen only one that build?  Meaning if you install the previous build, does it behave differently?

Does it happen in English or only in a different language?

**Comment by cvancar** [comment]
Hi, I found it. It is actually related to the changed settings in my
Android 11 Display.
Font size - Default and Display size - Small causing this issue in XDrip+
as I described.

I changed the Android 11 System Settings/Display/Display size back to
Default and then the size of the Extra status line in XDrip is back to
normal.

I did not noticed this specific issue on other phone with Android 12 so I
guess this is just fine.

Thanks
Petr


S pozdravem

Petr Čvančar
blog.gandalf.cz

**Reply by Navid200** [reply]
Thanks for describing the solution.
